src/clients/mocks/MockConfigStoreClient.ts

```diff
@@ -72,23 +73,22 @@ export class MockConfigStoreClient extends AcrossConfigStoreClient {
     // Ensure an array for every requested event exists, in the requested order.
     // All requested event types must be populated in the array (even if empty).
     const globalConfigUpdateTimes: number[] = [];
-    const _events: Event[][] = eventNames.map(() => []);
+    const _events: Log[][] = eventNames.map(() => []);
     for (const event of this.eventManager.getEvents().flat()) {
       const idx = eventNames.indexOf(event.event as string);
       if (idx !== -1) {
         _events[idx].push(event);
       }
 
       if (event.event === "UpdatedGlobalConfig") {
-        const block = await event.getBlock();
-        globalConfigUpdateTimes.push(block.timestamp);
+        globalConfigUpdateTimes.push(getCurrentTime());
```

Comment:
This doesn't look equivalent to what was deleted. Just want to verify that this is correct.

Reply:
I guess it doesn't matter since it is in mocks

Reply:
Good catch; the behaviour _should_ be ~equivalent because the old `event.getBlock()` just resolved to `Date.now() / 1000`.

![getBlock](https://github.com/user-attachments/assets/b1d37555-c3c7-42c5-b166-642a76e90ec6)